src/SFA.DAS.EmployerIncentives.Api/Controllers/ApplicationController.cs

```diff
@@ -24,11 +25,20 @@ public ApplicationController(IMediator mediator, ILogger<ApplicationController>
 
         [HttpPost]
         [Route("/accounts/{accountId}/applications")]
-        public async Task<IActionResult> PostApplication(CreateApplicationRequest request)
+        public async Task<IActionResult> CreateApplication(CreateApplicationRequest request)
         {
             var applicationId = await _mediator.Send(new CreateApplicationCommand(request.ApplicationId, request.AccountId, request.AccountLegalEntityId, request.ApprenticeshipIds));
 
-            return new CreatedResult($"/accounts/{request.AccountId}/applications/{request.ApplicationId}", null);
+            return new CreatedResult($"/accounts/{request.AccountId}/applications/{applicationId}", null);
+        }
+
+        [HttpPut]
+        [Route("/accounts/{accountId}/applications")]
+        public async Task<IActionResult> UpdateApplication(UpdateApplicationRequest request)
+        {
+            var applicationId = await _mediator.Send(new UpdateApplicationCommand(request.ApplicationId, request.AccountId, request.AccountLegalEntityId, request.ApprenticeshipIds));
```

Comment:
What's the point of returning the applicationId?  Same question would go for CreateApplication but I didn't review that one.